distributed/cli/tests/test_dask_scheduler.py

```diff
@@ -296,3 +296,64 @@ def check_scheduler():
                         c.scheduler.address
     finally:
         shutil.rmtree(tmpdir)
+
+PRELOAD_COMMAND_TEXT = """
+import click
+_config = {}
+
+@click.command()
+@click.option("--passthrough", type=str, default="default")
+def dask_command(passthrough):
+    _config["passthrough"] = passthrough
```

Comment:
I'm inclined to just put the command bits on `dask_setup` rather than add a new special function.  Presumably these run at the same time anyway?

Reply:
Understood, fixed below.